### PR TITLE
SONAR-22351 Calibrate startupProbe and other probes' initial delay

### PR DESCRIPTION
**Pull request by carminevassallo**

### CODE REVIEW

**Review by jCOTINEAU — APPROVED** [review]
lgtm, during scaling test, looks about right

**Review by jCOTINEAU — CHANGES_REQUESTED** [review]
Approved the PR conceptually, waiting for the usual helm changes for changelog and so on.

File: charts/sonarqube-dce/README.md [hunk 1]
```diff
@@ -221,17 +221,17 @@ The following table lists the configurable parameters of the SonarQube chart and
...
+| `searchNodes.livenessProbe.failureThreshold`              | LivenessProbe threshold for marking as dead                                            | `6`                                                                    |
 | `searchNodes.livenessProbe.timeoutSeconds`                | LivenessProbe timeout delay                                                           | `1`                                                                    |
-| `searchNodes.startupProbe.initialDelaySeconds`            | StartupProbe initial delay for Search Node checking                                   | `30`                                                                   |
+| `searchNodes.startupProbe.initialDelaySeconds`            | StartupProbe initial delay for Search Node checking                                   | `20`                                                                   |
 | `searchNodes.startupProbe.periodSeconds`                  | StartupProbe period between checking Search Node                                      | `10`                                                                   |
```
Comment (jCOTINEAU) [inline charts/sonarqube-dce/README.md:233]: searchNodes.startupProbe.periodSeconds looks like is at 5 instead of 10

**Review by jCOTINEAU — CHANGES_REQUESTED** [review]

File: charts/sonarqube-dce/README.md (hunk 1 above)
Comment (carminevassallo) [reply charts/sonarqube-dce/README.md:233]: Hey @jCOTINEAU, it's actually at 10 in the value file..

**Review by jCOTINEAU — APPROVED** [review]
lgtm !

**Pull request merged**